Source/inv.cpp

```diff
@@ -2070,7 +2070,10 @@ BOOL UseInvItem(int pnum, int cii)
 
 	if (plr[pnum]._pInvincible && !plr[pnum]._pHitPoints && pnum == myplr)
 		return TRUE;
-	if (pcurs != 1)
+
+	// Controller support: Allow using belt items without cursor.
+	// if (pcurs != 1)

```

Comment:
Leaving dead code like this with a comment was a bit of what was not so grate about the original code, it would be better to have a commit that makes the change and has the comment in as the commit message.

Reply:
I though it would be helpful when merging from devilution.
Should I remove all these lines?

Reply:
Its easier with a commit and where one line is clearly being swapped for another. The conflict will be minimal (because it's not a huge commit), and the commit message is shown as a heading to the conflict.

Reply:
Well they're not separate commits, but yeah the changes to `Source` in this implementation are minimal. Done.

Reply:
alright, I'm becoming an expert at rewriting history any way :)